uobtheatre/productions/schema.py

```diff
@@ -223,6 +225,9 @@ def resolve_total_tickets_sold(self, info):
     def resolve_content_warnings(self, info):
         return self.warnings_pivot.order_by("warning__short_description").all()
 
+    def resolve_shortdescription(self, info):
+        return self.shortdescription
+
```

Comment:
Can you try without these lines in this file (and lines 196-197)? According to James we shouldn't need (I'm not too sure but if we don't need it can get rid of it!)